README.md

```diff
@@ -33,67 +36,67 @@ Utilities for Insights Results Aggregator
         * [Generated documentation](#generated-documentation-2)
         * [Usage](#usage-2)
         * [A real example](#a-real-example)
-    * [`gen_broken_messages.py`](#gen_broken_messagespy)
+    * [`gen_messages.py`](#gen_messagespy)
         * [Description](#description-7)
         * [Generated documentation](#generated-documentation-3)
         * [Usage](#usage-3)
-    * [`gen_broken_jsons.py`](#gen_broken_jsonspy)
+    * [`gen_broken_messages.py`](#gen_broken_messagespy)
         * [Description](#description-8)
         * [Generated documentation](#generated-documentation-4)
         * [Usage](#usage-4)
-    * [`random_payload_generator.py`](#random_payload_generatorpy)
+    * [`gen_broken_jsons.py`](#gen_broken_jsonspy)
         * [Description](#description-9)
         * [Generated documentation](#generated-documentation-5)
         * [Usage](#usage-5)
-* [Utilitites for generating reports etc.](#utilitites-for-generating-reports-etc)
-    * [`stat.py`](#statpy)
+    * [`random_payload_generator.py`](#random_payload_generatorpy)
         * [Description](#description-10)
         * [Generated documentation](#generated-documentation-6)
         * [Usage](#usage-6)
-    * [`affected_clusters.py`](#affected_clusterspy)
+* [Utilitites for generating reports etc.](#utilitites-for-generating-reports-etc)
+    * [`stat.py`](#statpy)
         * [Description](#description-11)
         * [Generated documentation](#generated-documentation-7)
         * [Usage](#usage-7)
-    * [`reports.py`](#reportspy)
+    * [`affected_clusters.py`](#affected_clusterspy)
         * [Description](#description-12)
-        * [Generated documentatin](#generated-documentatin)
+        * [Generated documentation](#generated-documentation-8)
         * [Usage](#usage-8)
-    * [`cluster_results_age.py`](#cluster_results_agepy)
+    * [`reports.py`](#reportspy)
         * [Description](#description-13)
-        * [Generated documentatin](#generated-documentatin-1)
+        * [Generated documentatin](#generated-documentatin)
         * [Usage](#usage-9)
-* [Utilities for working with objects stored in AWS S3 bucket](#utilities-for-working-with-objects-stored-in-aws-s3-bucket)
-    * [`upload_timestamps.py`](#upload_timestampspy)
+    * [`cluster_results_age.py`](#cluster_results_agepy)
         * [Description](#description-14)
-        * [Generated documentation](#generated-documentation-8)
+        * [Generated documentatin](#generated-documentatin-1)
```

Comment:
where's the typo coming from?

Reply:
from text which is source of truth. fixed there, good catch